docs/developer-guide/interactivity.md

```diff
@@ -1,8 +1,72 @@
 # Adding Interactivity
 
-> This article discusses interacting with data (i.e. selecting, or picking objects). Viewport controls (panning, zooming etc) are discussed in [Controllers](/docs/api-reference/map-controller.md).
+## Controlling the camera
 
-## Overview
+In its basic form, deck.gl maintains its own camera state and the keyboard or mouse are used to pan/zoom/tilt layers along with any underlying map or imagery.
+
+To define the initial camera view, set the `initialViewState` property to define the initial camera position. 
+
+For more flexibility you can maintain the view state yourself and pass it in to deck.gl via the `viewState` parameter. This allows you to perform  fly to transitions to a new location, set pitch, zoom or any other property of the viewState.
+
+*Note : Do not combine initialViewState and viewState parameters*
+
+The following example demonstrates the viewState being controlled outside of deck.gl using a setInterval. it also demonstrates the ability to smoothly transition the camera by using the FlyToInterpolator. 
+
+### Using React
```

Comment:
This section is organized linearly - one has to read the full text to understand the context of this example. Most people scan through a page quickly by the headers. To provide more clarity, we need one subheader for each usage pattern:

- Controlling the camera
  + Basic form: automatically track camera updates
  + Stateless: externally controlled camera states